declaration-diagnosis-redesign/choosing-answer-strategy.qmd

```diff
@@ -7,57 +7,57 @@
 source("scripts/before_chapter_script.R")
 ```
 
-The answer strategy is a plan for what to do with the information gathered from the world in order to generate an answer to the inquiry. Qualitative and quantitative methods courses provide guidance about the properties of different strategies and the conditions under which they work well or poorly. Under what conditions should we use ordinary least squares, when should we use logit? When is a machine learning algorithm the appropriate choice and when would a comparative case study be more informative? When is *no* answer strategy worth pursuing because of the fundamental limitations of the data strategy?
+The answer strategy is a plan for what to do with the information gathered from the world (or from a model of the world) in order to generate an answer to the inquiry. Qualitative and quantitative methods courses provide guidance about the properties of different strategies and the conditions under which they work well or poorly. Under what conditions should we use ordinary least squares, when should we use logit? When is a machine learning algorithm the appropriate choice and when would a comparative case study be more informative? When is *no* answer strategy worth pursuing because of the fundamental limitations of the data strategy?
 
-Following @exm-designforpurpose: Design for purpose, the evaluation of an answer strategy depends on our ultimate goals: what is the answer to be used for? A perfect answer is generally elusive in empirical research so in practice we often need to select among strategies that come with different strengths and weaknesses. For instance some might suffer less from bias while others might be more precise. In other words, which answer strategy is best for you depends on what diagnosands you care about.
+Following @exm-designforpurpose: `Design for purpose`, the evaluation of an answer strategy depends on our ultimate goals: what is the answer to be used for? A perfect answer is generally elusive in empirical research, so in practice we often need to select among strategies that come with different strengths and weaknesses. For instance, some might suffer less from bias while others might be more precise. In other words, which answer strategy is best for you depends on what diagnosands you care about.
```

Comment:
this should be *Design for purpose* not back ticks. back ticks turn it into the code format. @williamfernandezt can you check for this throughout your edits (or maybe this was just early on?)

Reply:
I corrected this and now the text is in italics. I believe this was just a one-time mistake from my part.